kubernetes/cray-opa/values.yaml

```diff
@@ -56,15 +56,13 @@ opa:
   # increase this to 10s.
   httpTimeout: 10s
   requireHeartbeatToken: False
-  logPost: False
   xnamePolicy:
     enabled: True
     bos: True
     cfs: True
-    heartbeat: False
-    # ckdump and dvs needs logPost to be true in order to function properly
     ckdump: False
     dvs: False
+    heartbeat: False
```

Comment:
There should be OPA rego tests where heartbeat is set to True.

Reply:
xname-enabled tests set this to true and ingressgateway_policy_test.xname_workloads.rego.tpl and ingressgateway_policy_test.xname_workloads.invalid_xname.rego.tpl have tests to validate the heartbeat xname filtering works. (These are two separate files because ingressgateway_policy_test.xname_workloads.rego.tpl is used for both enabled and disabled xname tests, where as ingressgateway_policy_test.xname_workloads.invalid_xname.rego.tpl  is only run when xname validation is enabled)